scripts/start_civ.sh

```diff
@@ -362,6 +362,41 @@ function set_guest_time_keep() {
     fi
 }
 
+
+function setup_secure_factory_reset(){
+    tpm_binding=$SCRIPTS_DIR/tpm_binding
+	persistent_image=$WORK_DIR/persistent.img
+	data_image=$WORK_DIR/userdata.img
+
+	if [ ! -f "$persistent_image" ]; then
+		echo "create $persistent_image ...."
+		dd if=/dev/zero of=$persistent_image bs=1024 count=16
+	fi
+
+	if [ ! -f "$data_image" ]; then
+		echo "create $data_image ...."
+		dd if=/dev/zero of=$data_image bs=1M count=4096
+	fi
+
+	commandstr="\
+		-drive file=$data_image,if=none,id=disk2 \
+		-device virtio-blk-pci,drive=disk2,bootindex=2 \
+		-drive file=$persistent_image,if=none,id=disk3 \
+		-device virtio-blk-pci,drive=disk3,bootindex=3 \
+	"
+	GUEST_EXTRA_QCMD=${GUEST_EXTRA_QCMD}${commandstr}
```

Comment:
GUEST_EXTRA_QCMD is for scripts input usage. please define a new global variable to assign the commandstr. And append the variable to EXE_CMD.